src/test/java/org/opensearch/security/authtoken/jwt/JwtVendorTest.java

```diff
@@ -270,4 +272,103 @@ public void testCreateJwtLogsCorrectly() throws Exception {
         final String[] parts = logMessage.split("\\.");
         assertTrue(parts.length >= 3);
     }
+
+    @Test
+    public void testCreateJwtForApiTokenLogsCorrectly() throws Exception {
+        final String issuer = "cluster_0";
+        final String subject = "test-token";
+        final String audience = "test-token";
+        final List<String> clusterPermissions = List.of("cluster:admin/*");
+        ApiToken.IndexPermission indexPermission = new ApiToken.IndexPermission(List.of("*"), List.of("read"));
+        final List<ApiToken.IndexPermission> indexPermissions = List.of(indexPermission);
+        final String expectedClusterPermissions = "cluster:admin/*";
+        final String expectedIndexPermisions = indexPermission.toString();
+
+        LongSupplier currentTime = () -> (long) 100;
+        String claimsEncryptionKey = "1234567890123456";
+        Settings settings = Settings.builder()
+            .put("signing_key", signingKeyB64Encoded)
+            .put("encryption_key", claimsEncryptionKey)
+            // CS-SUPPRESS-SINGLE: RegexpSingleline get Extensions Settings
+            .put(ConfigConstants.EXTENSIONS_BWC_PLUGIN_MODE, true)
```

Comment:
why this setting? also we should add a test for unsupported algorithm exception?

Reply:
Removed this setting, it was unnecessary. The unsupported algorithm I got was from when my key was not long enough, when it is long enough it does not throw that exception. In any case I also added a test for it in the JWT vendor (which is where it originates from) in: [b84d2cf](https://github.com/opensearch-project/security/pull/4967/commits/b84d2cfb34369a11717d102fb04f27977b8d3510)